src/sieppari/core.cljc

```diff
@@ -1,31 +1,34 @@
 (ns sieppari.core
+  #?(:cljs (:refer-clojure :exclude [iter]))
   (:require [sieppari.queue :as q]
             [sieppari.async :as a]
-            [sieppari.async.ext-lib-support])
-  (:import (java.util Iterator)))
+            #?(:clj [sieppari.async.ext-lib-support]
+               :cljs [goog.iter :as iter]))
```

Comment:
I don't think I see this used anywhere in this namespace?

Reply:
Nice catch, will remove